src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AuditApiActionTest.java

```diff
@@ -52,6 +81,177 @@ public void testInvalidPath() throws Exception {
         assertEquals(HttpStatus.SC_METHOD_NOT_ALLOWED, response.getStatusCode());
     }
 
+    @Test
+    public void testDisabledCategoryOrder() throws Exception {
+        setup();
+
+        final List<String> testCategories = ImmutableList.of("SSL_EXCEPTION", "AUTHENTICATED", "BAD_HEADERS");
+        final AuditConfig auditConfig = new AuditConfig(true, AuditConfig.Filter.from(
+                ImmutableMap.of("disabled_rest_categories", testCategories)
+        ), ComplianceConfig.DEFAULT);
+        final ObjectNode json = DefaultObjectMapper.objectMapper.valueToTree(auditConfig);
+
+        testPutRequest(json, HttpStatus.SC_OK, true);
+        RestHelper.HttpResponse response = rh.executeGetRequest(ENDPOINT, adminCredsHeader);
+        List<String> actual = Streams.stream(readTree(response.getBody()).at("/config/audit/disabled_rest_categories").iterator())
+                .map(JsonNode::textValue)
+                .collect(Collectors.toList());
+        assertEquals(testCategories, actual);
+    }
+
+    @Test
+    public void testReadonlyApi() throws Exception {
+        final List<String> readonlyFields = ImmutableList.of("/audit/enable_rest", "/audit/disabled_rest_categories", "/audit/ignore_requests", "/compliance/read_watched_fields");
+        updateStaticResourceReadonly(readonlyFields);
```

Comment:
Had to resort to updating the test file in test resources as the resource file could not be injected into AuditApiAction for testing. 
In tearDown clearing it. 

Attempted to change via Reflection to inject another value for the `STATIC_RESOURCE` but making tests to be unstable